src/NativeScript/ObjC/ObjCPrototype.mm

```diff
@@ -60,20 +63,83 @@ static EncodedJSValue JSC_HOST_CALL getIterator(ExecState* execState) {
     }
 
     ObjCPrototype* prototype = jsCast<ObjCPrototype*>(object);
+    if (UNLIKELY(propertyName == prototype->_definingPropertyName)) {
+        // We're currently defining it, it's still not defined
+        return false;
+    }
+
+    GlobalObject* globalObject = jsCast<GlobalObject*>(prototype->globalObject());
+    // Check for property
+    if (auto propertyMeta = prototype->_metadata->instanceProperty(propertyName.publicName(), prototype->klass())) {
+        prototype->_definingPropertyName = propertyName;
+        prototype->defineNativeProperty(execState->vm(), globalObject, propertyMeta);
+        prototype->_definingPropertyName = PropertyName(nullptr);
+
+        //        std::chrono::time_point<std::chrono::system_clock> endTime = std::chrono::system_clock::now();
```

Comment:
It would be better if we extract this code, so that it can be used on other places as well instead of copy-pasting it